Cargo.toml

```diff
@@ -24,7 +24,7 @@ libc = { version = "0.2.58", optional = true }
 xenctrl = { git = "https://github.com/Wenzel/xenctrl", optional = true }
 xenstore = { git = "https://github.com/Wenzel/xenstore", optional = true }
 xenforeignmemory = { git = "https://github.com/Wenzel/xenforeignmemory", optional = true }
-kvmi = { git = "https://github.com/Wenzel/kvmi", optional = true }
+kvmi = { git = "https://github.com/Wenzel/kvmi", rev = "b9d8df6730b46f6a00c9ba6e1adcdd5434f613d7", optional = true }
```

Comment:
What happens if we use master branch ?
I don't want to commit a specific revision here.
Also, we should switch to published crates.

Reply:
Libmicrovmi is currently broken on master after the last kvmi merge. Why not pin a commit? It is functionally equivalent to pinning a version and we don't have published crates right now.

Reply:
I consider this a temporary solution until we either merge a PR that is compatible with the latest changes in kvmi or start publishing crates on crates.io

Reply:
now it makes sense